Runtime/Scripts/DeviceInfoIntegration.cs

```diff
@@ -174,9 +195,5 @@ enum NetworkDataType
   // Used for DeviceInfo in UnityEditor (any target platform)
   public class TestDeviceInfo : DeviceInfoIntegration
   {
-    public Dictionary<string, string> GetDeviceInfo()
```

Comment:
I think this might fail to compile. I think we need a GetDeviceInfo function in order to implement the DeviceInfoIntegration interface

Reply:
No, it doesn't fail to complie, Actually it's giving me an warning since DeviceInfoIntegration (the Parent) have a public GetDeviceInfo method which TestDeviceInfo() inherits.

Reply:
ah i think this class should implement DeviceInfo interface instead of inheriting from DeviceInfoIntegration. That way we can return dummy values from GetDeviceInfo